sdk/storage/azure-storage-blob-batch/README.md

```diff
@@ -0,0 +1,330 @@
+
+```java
+try(ByteArrayOutputStream outputStream = new ByteArrayOutputStream()) {
+    blobClient.download(outputStream);
+}
+```
+
+### Downloading a blob to local path
+
+Download blob to local file using BlobClient.
+```java
+blobClient.downloadToFile("downloaded-file.jpg");
+```
+
+### Enumerating blobs
+
+Enumerating all blobs using BlobContainerClient
+```java
+blobContainerClient.listBlobsFlat()
+        .forEach(
+            blobItem -> System.out.println("This is the blob name: " + blobItem.getName())
+        );
+```
+
+### Authenticate with Azure.Identity
+
+The [Azure Identity library][identity] provides Azure Active Directory support for authenticating with Azure Storage.
+
+```java
+BlobServiceClient storageClient = new BlobServiceClientBuilder()
+        .endpoint(endpoint)
+        .credential(new DefaultAzureCredentialBuilder().build())
+        .buildClient();
+```
+
+## Troubleshooting
+
+When interacts with blobs using this Java client library, errors returned by the service correspond to the same HTTP
+status codes returned for [REST API][error_codes] requests. For example, if you try to retrieve a container or blob that
+doesn't exist in your Storage Account, a `404` error is returned, indicating `Not Found`.
+
+## Next steps
+
+Get started with our [Blob samples][samples]:
+
+1. [Basic Examples][samples_basic]: Create storage, container, blob clients, Upload, download, and list blobs.
+1. [File Transfer Examples][samples_file_transfer]: Upload and download a large file through blobs.
+1. [Storage Error Examples][samples_storage_error]: Handle the exceptions from storage blob service side.
+1. [List Container Examples][samples_list_containers]: Create, list and delete containers.
+1. [Set metadata and HTTPHeaders Examples][samples_metadata]: Set metadata for container and blob, and set HTTPHeaders for blob.
+1. [Azure Identity Examples][samples_identity]: Use DefaultAzureCredential to do the authentication.
+
+## Contributing
+
+This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the Code of Conduct FAQ or contact opencode@microsoft.com with any additional questions or comments.
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-java%2Fsdk%2Fstorage%2FAzure.Storage.Blobs%2FREADME.png)
```

Comment:
```suggestion
![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-java%2Fsdk%2Fstorage%2FAzure.Storage.Blobs.Batch%2FREADME.png)
```

Reply:
Delete this line in favor of the impression at the bottom of the file, should make sure if both are needed later.